### DISCUSSION TITLE
Wrong/confusing `focus the "to" input field` comment in example

### DISCUSSION THREAD

**Discussion by jnachtigall** [gpbl/react-day-picker#1161] [Support]
I am new to react-day-picker, sorry, if this is the wrong place to report. Could not find any other place.

On http://react-day-picker.js.org/examples/input-from-to there is:
```js
  handleFromChange(from) {
    // Change the from date and focus the "to" input field
    this.setState({ from });
  }
```
The `and focus the "to" input field` comment imho makes no sense here. There's no focus being set here. 

The next function does something like setting the `from`'s month (but still not focus):
```js
  handleToChange(to) {
    this.setState({ to }, this.showFromMonth);
  }
```

This confused me. (And I could not find the source for it to try to improve.)

**Comment by KyorCode** [comment]
That comment is indeed confusing. but what happens is:

- Day Click in from 
- handleFromChange trigger
- onDayClicked trigger

Check the DayPickerProps on the 'from' DayPickerInput. It states: `this.to.getInput().focus()`
